src/solver/opamCudf.ml

```diff
@@ -969,6 +1026,9 @@ let extract_explanations packages cudfnv2opam reasons : explanation list =
         | _ -> false)
   in
   match explanations with
+  | [] ->
+    OpamConsole.error_and_exit `Internal_error
+      "Internal error while computing conflict explanations: sorry, please report."
```

Comment:
In general a conflict is the end of story for opam commands... still, it feels weird to hard error out from a function which sole purpose is to print an informative message. Does returning the "internal error" code really matter here, or couldn't we print the error message but just return `[]` ?

Reply:
Note: this is part of https://github.com/ocaml/opam/pull/4981.

Mmh, we could make it into `OpamConsole.error` instead but this would make opam print two different user direction:
```
[ERROR] Package conflict!
[ERROR] Internal error while computing conflict explanations: sorry, please report.
Sorry, no solution found: there seems to be a problem with your request.
```
I’d rather push users to report instead of making them try to fiddle with the request and maybe give up reporting.